website/src/views/h1n1pdm.ts

```diff
@@ -0,0 +1,177 @@
+import { dateRangeOptionPresets, type MutationAnnotation } from '@genspectrum/dashboard-components/util';
+
+import { type CompareSideBySideData, type DatasetAndVariantData, type Id } from './View.ts';
+import type { OrganismsConfig } from '../config.ts';
+import {
+    BaseView,
+    GenericCompareToBaselineView,
+    GenericCompareVariantsView,
+    GenericSequencingEffortsView,
+    GenericSingleVariantView,
+} from './BaseView.ts';
+import { type OrganismConstants, getAuthorRelatedSequencingEffortsFields } from './OrganismConstants.ts';
+import { compareSideBySideViewConstants } from './ViewConstants.ts';
+import type { BaselineFilterConfig } from '../components/pageStateSelectors/BaselineSelector.tsx';
+import type { LineageFilterConfig } from '../components/pageStateSelectors/LineageFilterInput.tsx';
+import { organismConfig, Organisms } from '../types/Organism.ts';
+import { type DataOrigin, dataOrigins } from '../types/dataOrigins.ts';
+import { CompareSideBySideStateHandler } from './pageStateHandlers/CompareSideBySidePageStateHandler.ts';
+
+const earliestDate = '1900-01-01';
+const hostField = 'hostNameScientific';
+
+class H1n1pdmConstants implements OrganismConstants {
+    public readonly organism = Organisms.h1n1pdm;
+    public readonly earliestDate = earliestDate;
+    public readonly mainDateField: string;
+    public readonly locationFields: string[];
+    public readonly lineageFilters: LineageFilterConfig[] = [
+        {
+            lapisField: 'cladeHA',
+            placeholderText: 'Clade HA',
+            filterType: 'text' as const,
+            initialValue: undefined,
```

Comment:
Could you please rebase? After I merged #606, this should not be necessary anymore (and give you a type error).